boards/BMS/Inc/App/App_CurrentSense.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "App_ErrorCode.h"
+
+/**
+ * Convert the given ADC voltage to main current 1
+ * @param adc_voltage The ADC voltage to convert
+ * @param main_current_1 This will be set to main current 1, in amps
+ * @return EXIT_CODE_OUT_OF_RANGE if adc_voltage is negative
+ *         EXIT_CODE_INVALID_ARGS if main_current_1 is NULL
+ */
+ExitCode App_CurrentSense_ConvertToMainCurrent1(
+    float  adc_voltage,
+    float *main_current_1);
```

Comment:
I'm a bit confused about what MainCurrent1 is? Is this dictacted by hardware?

Reply:
They're names carried over from the hardware schematic. Basically we call the current in the high voltage path (i.e. 400V battery pack) "Main current". The current sensor for this has two outputs (for two different resolutions), hence, Main Current 1 and Main Current 2.

![image](https://user-images.githubusercontent.com/16970019/82134144-64861f00-97a9-11ea-8c94-bc8b9f013628.png)

Reply:
The names aren't great. But since I'm not using these currents in the state machine anywhere yet it's hard to come up with a better name. So I thought i'd just leave it the same as the schematic for now.

Reply:
If the difference between the two is the different resolution, perhaps we could make that part of the name?

Reply:
Yup that's fine with me